### PR TITLE
Prevent PHP CS scanning stray folders

### LINKED ISSUE #31863: PHP code style check is checking data folder and apps2 and apps3 folders
### Steps to reproduce
1. install development ownCloud in local git repo with the data folder in ``data`` of the local repo
2. as some user, upload some *.php code with crap style
3. ``make test-php-style``

### Expected behaviour
Style checks pass

### Actual behaviour
Style check fails on the file(s) that are actually just test user data, not real ownCloud code

### Server configuration
Local development environment.

### PR DESCRIPTION
**Pull request by PVince81**
When swiching from stable9.1 the "3rdparty" folder can remain in some
cases.
Also exluded "tests/autoconfig*" which are sometimes generated when
running unit tests.

Fixes https://github.com/owncloud/core/issues/31863

Any additions must be made to ".php_cs" folder as I did for my own setup with "apps2" and "apps3"

### CODE REVIEW

File: .php_cs.dist [hunk 1]
```diff
@@ -36,8 +37,9 @@ foreach ($dirIterator as $fileinfo) {
...
 $finder = PhpCsFixer\Finder::create()
     ->exclude($excludeDirs)
+    ->notPath('tests/autoconfig*')
     ->in(__DIR__);
 
 $config = new OC\CodingStandard\Config();
 $config->setFinder($finder);
-return $config;
\ No newline at end of file
+return $config;
```
Comment (patrickjahns) [inline .php_cs.dist]: newline?

Comment (PVince81) [reply .php_cs.dist]: We have a newline in all other PHP files so I think it's ok to have it here too for consistency.

Do you mind if I also convert all spaces to tabs as all OC's source files have tabs ?

**Comment by mmattel** [comment]
Maybe important, we already have in documentation to use the name apps-external instead of apps2.
There is also an open PR that creates that directory and sets the config.php parameter on new installs.
#30889 (Add per default the apps-external directory in config.php during installation)

**Comment by PVince81** [comment]
pushed another commit to convert the spaces to tabs, aligns the code style of this file with the rest of the project

**Comment by PVince81** [comment]
- [ ] also add "apps-external"

**Comment by patrickjahns** [comment]
🤔 Instead of excluding everything - we can go the other way around and specify explicit folders, files to include?

**Comment by PVince81** [comment]
sounds good, I'll look into this

**Review by phil-davis — APPROVED** [review]
This looks like a good thing to me - adds a couple more exclusions and does whitespace-tab changes.

**Comment by phil-davis** [comment]
@PVince81 are you going to add apps-external or not?

**Comment by PVince81** [comment]
well, I actually intended to look at inclusions... but as I'm already busy with lots of other things let's move on with exclusions now

I'll add apps-external

**Comment by PVince81** [comment]
added

**Comment by phil-davis** [comment]
Looks good to me. Refactoring to reverse the exclude/include process can always happen in a later PR if someone thinks it is worthwhile.

**Pull request merged**